views/partials/navbar.handlebars

```diff
@@ -0,0 +1,13 @@
+<link rel="stylesheet" href="static/css/navbar.css">
+
+<div id="navbar">
+  <a id="navbar-logo" href="/">
+    DONATIA
+  </a>
+
+  <div id="navbar-links">
```

Comment:
nit: to make our HTML more semantic you could change this div to use the `<nav>` tag since these links are navigational links. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav